docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+
+:Wrong: ``process_name:chrome.exe AND 192.168.1.1``
+:Right: ``process_name:chrome.exe AND netconn_ipv4:192.168.1.1``
+
+* Query IOCs may search on CIDR address ranges, e.g.: ``netconn_ipv4:192.168.0.0/16``.
+
+Feeds
+-----
+Another way of managing reports is to attach them to a *feed.* Feeds can contain multiple reports, and a feed can be
+attached to a watchlist, effectively making the contents of the watchlist equivalent to the contents of the feed.
+
+A new feed may be created as follows:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Feed, Report
+    >>> report = Report(api, None, {"description": "Unsigned processes impersonating browsers",
+    ...                             "iocs_v2": [{
+    ...                                 "id": "unsigned-chrome",
+    ...                                 "match_type": "query",
+    ...                                 "values": [("process_name:chrome.exe NOT process_publisher_state:FILE_SIGNATURE_STATE_SIGNED")]
+    ...                                 }],
+    ...                             "link": None,
+    ...                             "severity": 5,
+    ...                             "tags": ["compliance", "unsigned browsers"],
+    ...                             "timestamp": 1601326338,
+    ...                             "title": "Unsigned Browsers"})
+    >>> feed_data = {'feedinfo': {
+    ...                 'name': 'Suspicious Applications',
+    ...                 'provider_url': 'http://example.com/location',
+    ...                 'summary': 'Any signs of suspicious applications running on our endpoints',
+    ...                 'category': 'external_threat_intel',
+    ...                 'source_label': 'Where the info is coming from'},
+    ...              'reports': [report._info]}
+    >>> feed = api.create(Feed, feed_data)
+
+If you have an existing feed, a new report may be added to it as follows:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Feed, Report
+    >>> feed = cb.select(Feed, 'ABCDEFGHIJKLMNOPQRSTUVWX')
+    >>> report = Report(api, None, {"description": "Unsigned processes impersonating browsers",
+    ...                             "iocs_v2": [{
+    ...                                 "id": "unsigned-chrome",
+    ...                                 "match_type": "query",
+    ...                                 "values": [("process_name:chrome.exe NOT process_publisher_state:FILE_SIGNATURE_STATE_SIGNED")]
+    ...                                 }],
+    ...                             "link": None,
+    ...                             "severity": 5,
+    ...                             "tags": ["compliance", "unsigned browsers"],
+    ...                             "timestamp": 1601326338,
+    ...                             "title": "Unsigned Browsers"})
+    >>> feed.append_reports([report])
+
+**Note:** These feeds are *private feeds,* meaning they are only visible within an organization and can be created by
```

Comment:
This should be moved up from a note to the summary

Reply:
I moved it closer to the top of the section and had to rewrite it a teensy bit.